src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/client/ProfessionalApiClient.java

```diff
@@ -410,10 +411,17 @@ public void retrieveBadRequestForPendingOrganisationWithPbaEmail(String email, S
 
 
     @SuppressWarnings("unchecked")
-    public Map<String, Object> searchUsersByOrganisation(String organisationId, String role, String showDeleted, HttpStatus status) {
+    public Map<String, Object> searchUsersByOrganisation(String organisationId, String role, String showDeleted, HttpStatus status, String roleRequired) {

```

Comment:
please rename `rolesRequired` to the params actual name of `returnRoles`

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-professional-api/pull/612/commits/acbe4881fa2e5d46be921bc81e0a93e8f8bf9f4f